### PR TITLE
Rename MixedRealityToolkit.Services folder to MRTK.Services

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change is part one of a series of changes that will result in shorter paths (by 15 characters) for most of the MRTK (MixedRealityToolkit.Generated is not currently planned for a rename).

This change updates the Services code folder and related documentation references.

All test automation was run prior to opening this PR.

Related to #6985

### CODE REVIEW

File: Documentation/Architecture/InputSystem/CoreSystem.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Core system
 
-At the heart of the input system is the [MixedRealityInputSystem](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs),
+At the heart of the input system is the [MixedRealityInputSystem](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MRTK.Services/InputSystem/MixedRealityInputSystem.cs),
```
Comment (keveleigh) [inline Documentation/Architecture/InputSystem/CoreSystem.md]: Should most of these just be updated to `xref`s?

Comment (davidkline-ms) [reply Documentation/Architecture/InputSystem/CoreSystem.md]: oh, now that is a good thought

Comment (davidkline-ms) [reply Documentation/Architecture/InputSystem/CoreSystem.md]: wonder why they were blob links originally

File: Documentation/Architecture/InputSystem/CoreSystem.md [hunk 2]
```diff
@@ -37,9 +33,6 @@ This object would get touch events when focused by a hand that is near it. These
 events go "up" the GameObject hierarchy until it finds a GameObject that is capable of handling
 the event.
 
-This is done by using [ExecuteHierarchy](https://docs.unity3d.com/ScriptReference/EventSystems.ExecuteEvents.ExecuteHierarchy.html)
```
Comment (keveleigh) [inline Documentation/Architecture/InputSystem/CoreSystem.md]: Might be good to leave the `ExecuteHierarchy` link in, to add more context on how / why this event system works the way it does.

File: Documentation/README_ManipulationHandler.md [hunk 3]
```diff
@@ -6,9 +6,14 @@ The *ManipulationHandler* script allows for an object to be made movable, scalab
 
 ## How to use the manipulation handler
 
-Add the [`ManipulationHandler.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs) component to a GameObject.
+Add the `ManipulationHandler`script component to a GameObject:
```
Comment (keveleigh) [inline Documentation/README_ManipulationHandler.md]:
```suggestion
Add the `ManipulationHandler` script component to a GameObject:
```

File: Documentation/Architecture/InputSystem/CoreSystem.md (hunk 1 above)
Comment (davidkline-ms) [reply Documentation/Architecture/InputSystem/CoreSystem.md]: done!

File: Documentation/Architecture/InputSystem/CoreSystem.md (hunk 2 above)
Comment (davidkline-ms) [reply Documentation/Architecture/InputSystem/CoreSystem.md]: will restore. this is one instance where a link to the blob is most "reasonable"?

Comment (davidkline-ms) [reply Documentation/Architecture/InputSystem/CoreSystem.md]: done

File: Documentation/README_ObjectManipulator.md [hunk 4]
```diff
@@ -8,9 +8,14 @@ The *ObjectManipulator* script makes an object movable, scalable, and rotatable
 
 ## How to use the object manipulator
 
-To use the object manipulator, first add the [`ObjectManipulator.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Experimental/Features/Input/Handlers/ObjectManipulator.cs) component to a GameObject.
+To use the object manipulator, first add the `ObjectManipulator` script component to a GameObject:
 
-Make sure to also add a collider to the object, matching its grabbable bounds. To make the object respond to near articulated hand input, add the [`NearInteractionGrabbable.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionGrabbable.cs) script as well.
+- Selecting the GameObject in the Hierarchy
```
Comment (thalbern) [inline Documentation/README_ObjectManipulator.md]: do we really need those additional steps that describe how to add a component to a gameobject ? i think users should know that or look that up in the unity docs if they don't know - else we would have to put this in all our doc pages to stay consistent

File: Documentation/README_ManipulationHandler.md [hunk 5]
```diff
@@ -6,9 +6,14 @@ The *ManipulationHandler* script allows for an object to be made movable, scalab
 
 ## How to use the manipulation handler
 
-Add the [`ManipulationHandler.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs) component to a GameObject.
+Add the `ManipulationHandler` script component to a GameObject:
 
-Make sure to also add a collider to the object, matching its grabbable bounds. To make the object respond to near articulated hand input, add the [`NearInteractionGrabbable.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionGrabbable.cs) script as well.
+- Selecting the GameObject in the Hierarchy
+- In the Inspector, click the Add Component button
```
Comment (thalbern) [inline Documentation/README_ManipulationHandler.md]: same thing here - do we really need to explain how to add a component?

File: Documentation/README_ObjectManipulator.md (hunk 4 above)
Comment (davidkline-ms) [reply Documentation/README_ObjectManipulator.md]: This was a targeted update added to address the rename of the services folder. I was planning on putting similar instructions to other docs as I rename other folders.

I am happy to remove these or leave as is.

Comment (davidkline-ms) [reply Documentation/README_ObjectManipulator.md]: removed the additional steps.

File: Documentation/README_ManipulationHandler.md (hunk 5 above)
Comment (davidkline-ms) [reply Documentation/README_ManipulationHandler.md]: removed

**Pull request merged**